cmd/k8s-operator/deploy/chart/templates/oauth-secret.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Tailscale Inc & AUTHORS
+# SPDX-License-Identifier: BSD-3-Clause
+
+{{ if and .Values.oauth .Values.oauth.clientId -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: operator-oauth
```

Comment:
namespace

Reply:
Ahh I didn't realize we actually set namespace on each object. We didn't need to, it automatically adds namespace passed via `--namespace` flag to namespaced resources

Reply:
However, I also don't see any harm from adding it- will add for consistency